apps/vercel/frontend/src/locations/ConfigScreen.tsx

```diff
@@ -109,26 +110,33 @@ const ConfigScreen = () => {
     async function getApiPaths() {
       setIsLoading(true);
       if (vercelClient) {
-        const data = await vercelClient.listApiPaths(parameters.selectedProject);
-        setApiPaths(data || []);
+        try {
+          const data = await vercelClient.listApiPaths(parameters.selectedProject);
+          setApiPaths(validateApiPathData(data) ? data : []);
+        } catch (e) {
+          console.error(e);
+          setApiPaths([]);
```

Comment:
the `apiPaths` data will be an empty array if there is an error or if the data is not "valid"

Reply:
See below. I _think_ we might want to throw an error here and then catch and it handle upstream